functests/sctp/sctp_sriov.go

```diff
@@ -0,0 +1,202 @@
+package sctp
+
+import (
+	"context"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gstruct"
+
+	netattdefv1 "github.com/openshift/sriov-network-operator/pkg/apis/k8s/v1"
+	sriovv1 "github.com/openshift/sriov-network-operator/pkg/apis/sriovnetwork/v1"
+	sriovtestclient "github.com/openshift/sriov-network-operator/test/util/client"
+	sriovcluster "github.com/openshift/sriov-network-operator/test/util/cluster"
+	sriovdiscovery "github.com/openshift/sriov-network-operator/test/util/discovery"
+	sriovnamespaces "github.com/openshift/sriov-network-operator/test/util/namespaces"
+	sriovnetwork "github.com/openshift/sriov-network-operator/test/util/network"
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/openshift-kni/cnf-features-deploy/functests/utils/client"
+	"github.com/openshift-kni/cnf-features-deploy/functests/utils/discovery"
+	"github.com/openshift-kni/cnf-features-deploy/functests/utils/execute"
+	"github.com/openshift-kni/cnf-features-deploy/functests/utils/namespaces"
+	"github.com/openshift-kni/cnf-features-deploy/functests/utils/sriov"
+)
+
+const (
+	sriovOperatorNamespace = "openshift-sriov-network-operator"
+	testNetwork            = "test-sctp-sriov-network"
+)
+
+var _ = Describe("[sriov] SCTP integration", func() {
+	sriovclient := sriovtestclient.New("")
+	discoveryFailed := false
+	var testNode string
+
+	execute.BeforeAll(func() {
+		err := namespaces.Create(TestNamespace, client.Client)
+		Expect(err).ToNot(HaveOccurred())
+
+		Eventually(func() error {
+			_, err := client.Client.ServiceAccounts(TestNamespace).Get(context.Background(), "default", metav1.GetOptions{})
+			return err
+		}, 5*time.Minute, 5*time.Second).Should(Not(HaveOccurred()))
```

Comment:
I think 5 min here is too long can we change this to 1 min?

Reply:
This is the same as sctp, and the happy path is that it should be immediate, but I can change.